mne/io/utils.py

```diff
@@ -231,6 +231,7 @@ def _synthesize_stim_channel(events, n_samples):
     events : array, shape (n_events, 3)
         Each row representing an event as (onset, duration, trigger) sequence
         (the format returned by `_read_vmrk_events` or `_read_eeglab_events`).
+        Durations == 0 are converted to 1.
```

Comment:
Durations equal to 0 are converted to 1.

Reply:
not addressed yet